wasmer/types.go

```diff
@@ -33,6 +34,8 @@ func convertFromGoType(t reflect.Type) *wasmerGo.ValueType {
 		return wasmerGo.NewValueType(wasmerGo.F32)
 	case reflect.Float64:
 		return wasmerGo.NewValueType(wasmerGo.F64)
+	default:
+		log.DefaultLogger.Errorf("[wasmer][type] convertFromGoType unsupported type: %v", t.Kind().String())
```

Comment:
all the log in this PR are from mosn/mosn. No reason to keep a separate copy to do logging